boards/st/stm32n6570_dk/twister.yaml

```diff
@@ -14,6 +14,7 @@ supported:
   - i2c
   - gpio
   - memc
+  - sdmmc
```

Comment:
Drop this- the board does not support SDMMC tests because ST's SDMMC driver uses the disk subsystem.

Reply:
Done